src/Prometheus.Client.AspNetCore/PrometheusExtensions.cs

```diff
@@ -25,6 +25,9 @@ public static IApplicationBuilder UsePrometheusServer(this IApplicationBuilder a
         public static IApplicationBuilder UsePrometheusServer(this IApplicationBuilder app, Action<PrometheusOptions> setupOptions)
         {
             var options = new PrometheusOptions();
+            options.CollectorRegistryInstance
+                = (ICollectorRegistry)app.ApplicationServices.GetService(typeof(ICollectorRegistry)) ?? Metrics.DefaultCollectorRegistry;
```

Comment:
`=`? Or maybe ` ??=`?

Reply:
Null condition does not make any sense here, as CollectorRegistryInstance property is null for sure at this point